### ISSUE TITLE
Ly no longer works

### ISSUE THREAD

**Issue by plasmoduck** [fairyglade/ly#296]
I havent used my laptop for a few months, however I recently turned it on and done some software updates (I use FreeBSD) and after rebooting discovered ly no longer works. When I type in my password and hit enter, it doesn't login to shell or start my .xinitrc, it just goes back to the ly screen and says User logged out. I cannot figure it out. Please help.

**Comment by ojsl1** [comment]
Same issue on my FreeBSD VM with herbstluft 0.9.2 installed through ports. Works fine on my host machine, but it's running Arch and is on 0.9.3. Something wrong with the BSD port?

**Comment by ii-Python** [comment]
Have you tried logging into shell and reinstalling ly?

**Comment by plasmoduck** [comment]
Yes I have uninstalled it and reinstalled it though pkg, even compiled it through ports, no luck

**Comment by plasmoduck** [comment]
> Same issue on my FreeBSD VM with herbstluft 0.9.2 installed through ports. Works fine on my host machine, but it's running Arch and is on 0.9.3. Something wrong with the BSD port?

You should file a bug report on https://bugs.freebsd.org/bugzilla/ so the port maintainers can fix it. I have but the more reports the more likely it will be fixed sooner than later. 😉

**Comment by plasmoduck** [comment]
> Same issue on my FreeBSD VM with herbstluft 0.9.2 installed through ports. Works fine on my host machine, but it's running Arch and is on 0.9.3. Something wrong with the BSD port?

I found that installing a previous version from /var/cache/pkg using ‘pkg add’ worked for now.
